src/modules/sprungbrett/reducers/setSprungbrettUrl.js

```diff
@@ -0,0 +1,9 @@
+import { handleAction } from 'redux-actions'
+import setSprungbrettUrl from '../actions/setSprungbrettUrl'
+
+/**
+ * The reducer to store the url for the sprungbrett extra
```

Comment:
Maybe add, that we need to do that in order to have it available for the fetcher

Reply:
done